colabs/prompts/W&B_LLM_QA_Bot.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    run = wandb.init(project=project_name, entity=entity, job_type="dataset", config=asdict(config))
</pre><p>In general instead of using the WandbTracer from wandb I meant to say we should import it from Langchain where the latest version currently lives:<a href="https://github.com/langchain-ai/langchain/blob/303724980cd97c8156bf516fba3f367451315bb2/libs/langchain/langchain/callbacks/tracers/wandb.py#L414" target="_blank">https://github.com/langchain-ai/langchain/blob/303724980cd97c8156bf516fba3f367451315bb2/libs/langchain/langchain/callbacks/tracers/wandb.py#L414</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>